VERSION_OMICRON

```diff
@@ -1 +1 @@
-8a40bb8
\ No newline at end of file
+0dad016
```

Comment:
Does this commit have the final implementation of anti-affinity going out in the next release? I see the previous commit generated some functions that were now removed.

Reply:
Yep! These were marked as hidden recently due to them not being quite ready yet https://github.com/oxidecomputer/omicron/pull/7878